charts/flowker/templates/configmap.yaml

```
@@ -53,6 +65,31 @@ data:
   OTEL_EXPORTER_OTLP_ENDPOINT_PORT: {{ .Values.flowker.configmap.OTEL_EXPORTER_OTLP_ENDPOINT_PORT | default "4317" | quote }}
   OTEL_EXPORTER_OTLP_ENDPOINT: {{ .Values.flowker.configmap.OTEL_EXPORTER_OTLP_ENDPOINT | default "" | quote }}
 
+  # Audit Database (PostgreSQL — required when MULTI_TENANT_ENABLED=false)
+  AUDIT_DB_HOST: {{ .Values.flowker.configmap.AUDIT_DB_HOST | default "" | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)